### DISCUSSION TITLE
blank screen with RTX with popOS

### DISCUSSION THREAD

**Discussion by shimodii** [i3/i3#5335] [Q&A]
hello,
when i change my default graphic card to Nvidia and start i3 session it shows a blank screen and even no mouse is appeared.
also its working with integrated Intel graphic card with no problems.

any idea !?

**Comment by okraits** [comment]
i3 uses the RandR outputs which are available. It seems your Xorg config using the nvidia card does not result in an usable output so there is no output to be used by i3.

**Reply by hexwell** [reply]
I received a mention notification from GitHub, from this answer. But there is no mention and I never interacted with this thread, strange...

**Reply by Airblader** [reply]
There was a spam bot attack on our repository last night and they probably tagged you in that post.

**Reply by hexwell** [reply]
Makes sense, thanks for the information

**Comment by neg-serg** [comment]
Looks quiet strange. I am RTX(3080ti) user, no any problems. Are you using notebook?

Try this:
1) xrandr --auto in ~/.xinitrc or i3 startup
2) blacklist nouveau
3) add nomodeset option to linux kernel
